lms/static/js/dashboard/track_events.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Track interaction with the student dashboard..
+ */
+
+var edx = edx || {};
+
+(function ($) {
+    'use strict';
+
+    edx.dashboard = edx.dashboard || {};
+
+    edx.dashboard.TrackEvents = function() {
+
+        // Emit an event when the "course title link" is clicked.
+        $(".course-title-link").on("click", function (event) {
+            var courseKey = $(event.target).data("course-key");
+            window.analytics.track(
```

Comment:
Did you try using `trackLink()`? If so and you had problems, what went wrong and what did you try doing to fix it? Like I said earlier, use of `track()` doesn't guarantee the events we're interested in will be emitted.